### ISSUE TITLE
Basic app complete

### ISSUE THREAD

**Issue by sanwong15** [sanwong15/AwesomeToDoAndroidApp#2]
Hi, the basic features of the app is complete. I am gonna keep working on it and put on more features. Any comments would be appreciated. Thanks
/cc @codepathreview @codepath

**Comment by nesquena** [comment]
Hi Hong San,

Thanks for applying and submitting your pre-work. Based on your LinkedIn it is unclear if you meet the selection criteria for  participating in the course. In particular:
- 1+ years of professional software development experience in a single role (shipping code to production and collaborating on a team strongly preferred)
- Bachelors or Masters Degree in CS/Engineering (OR additional 2+ years of professional experience)

Can you review that and let me know if you can confirm that you believe you meet the requirements? 

If not, I can still add you to our remote self-paced observer group so you will still have access to the curriculum and can learn at your pace. In the meantime, you should also check out the [resources we provide here as well](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-dont-have-much-or-any-professional-software-development-experience-how-can-codepath-help-me). Thanks for your understanding.

**Comment by sanwong15** [comment]
Hi Nathan,

Thanks for reaching out to me.  I hold a Master Degree in Electrical Engineering specialized in Machine Learning and computer vision. I was a summer R&D intern at Mipowed Limited and an Engineering student helper at Delphi Systems for a quarter. Hope that answer your questions. Am I still allowed to update/modify my project before June 24?

Thanks

Hong San Wong
M.S Electrical Engineering 15”
University of California, Irvine
hswong1@uci.edu

**Comment by nesquena** [comment]
> Am I still allowed to update/modify my project before June 24?

Yes, definitely please do and thanks for the additional details. 

If you are not selected as a participant, we will be happy to add you to our remote self-paced observer group so you will still have access to the curriculum and can learn at your pace.

**Comment by sanwong15** [comment]
Thank you. I will definitely love to be considered as a participant. Thank you so much

Hong San Wong
M.S Electrical Engineering 15”
University of California, Irvine
hswong1@uci.edu
